WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduler.swift

```diff
@@ -160,31 +160,44 @@ class BloggingRemindersScheduler {
 
     private static func copyStoreToSharedFile() {
         guard let store = try? defaultStore(),
-              let fileUrl = try? defaultDataFileURL(),
               let sharedFileUrl = sharedDataFileURL() else {
             return
         }
 
-        // Only copy the file if we have at least one reminder schedule
-        if store.configuration.count > 0 {
-            try? FileManager.default.copyItem(at: fileUrl, to: sharedFileUrl)
+        var configuration = [String: ScheduledReminders]()
+        for (blogIdentifier, schedule) in store.configuration {
+            guard let objectID = ContextManager.shared.mainContext.persistentStoreCoordinator?.managedObjectID(forURIRepresentation: blogIdentifier),
+                  let blog = ContextManager.shared.mainContext.object(with: objectID) as? Blog,
```

Comment:
This method may be [called from the background queue](https://github.com/wordpress-mobile/WordPress-iOS/blob/release/21.3/WordPress/Classes/System/WordPressAppDelegate.swift#L330-L334) during app launch. 

Will this cause thread access problems since the `mainContext` is `viewContext` meant to be accessed from the main queue?

Reply:
That's a good call to move it to a background context. I tested it while on a background thread using the `mainContext` and it appeared to work correctly. Maybe just reading on a background thread doesn't cause any obvious issues? There's probably still a potential data issue though.

Reply:
> Maybe just reading on a background thread doesn't cause any obvious issues

That could be the case, yeah. Or that the DispatchQueue happened to use the main queue. 😂  Anyways, I think it's looking good. 👍🏼